projects/driver_controls/src/pedal_calibration.c

```diff
@@ -0,0 +1,192 @@
+// The goal of calibration is to provide an initialized ThrottleCalibrationData storage for the
```

Comment:
Nit: Separate this with extra newlines into paragraphs. It will just be easier to read that way.